### ISSUE TITLE
【Pytorch MLP】高层接口 类API与函数API

### ISSUE THREAD

**Issue by ShichaoJin** [ShichaoJin/tools#7]
## inplace 操作减少内存
x = layer1(x)
x = F.relu(x, inplace= True)   inplace可减少内存

class MLP(nn.Module):
def __init__(self)
super(MLP, self).__init__()
self.model = nn.Sequential(nn.Linear(), nn,Relu(inplace=True), )

def forward(self, x):
x = self.model(x)
return x

# 类风格的API  nn.Relu需要先实例化  v.s. 函数风格的APIF.relu() 
layer = nn.Relu()
x = layer(x)

x = F.relu(x, inplace=True)

## 
net = MLP()  模型实例化
optimizer = optim.SGD(net.parameters(), lr = learningrate)
criterion= nn.CrossEntropyLoss()
for ..
logits = net(data)
loss = criterion(logits, target)
